docs/content/getting-started/developers.mdx

```diff
@@ -56,19 +56,20 @@ There are a few helpers you may want to use within Circuit.
 Sometimes you may need to apply global styles to your page. Here is how you can do this with the utilities provided by Circuit UI.
 
 ```javascript
-import { theme, injectGlobalStyles } from '@sumup/circuit-ui';
+import { GlobalStyles } from '@sumup/circuit-ui';
+import { css } from '@emotion/core';
 
 const customStyles = `
   body {
     color: red;
   }
 `;
 
-injectGlobalStyles({ theme: theme.standard, custom: customStyles });
+<GlobalStyles custom={globalStyles}/>
```

Comment:
@fernandofleury Is this accurate? Just to double check, we prefer people to use `GlobalStyles` rather than emotion's normally exposed `Global` component?

Reply:
I'm not Fernando, but yes. The `GlobalStyles` component includes the reset styles that Circuit UI needs to work properly.

Reply:
OK, but the `<GlobalStyles>` is the general global styles, and if someone wants to provide custom global styles, do they use `<Global>` or the `custom` prop here? Which is the "best practice" for us?

Reply:
Ah! So, you should include the `<GlobalStyles>` component once in your application. You can pass it any other global, application level styles. If you have a component that needs to add global styles, it's better to use `<Global>`.

Reply:
In this case, what's the purpose of the `custom` prop then? Just want to write up something to explain this but it's not clear to me why we accept additional custom global styles in this component.

Reply:
> You can pass it any other global, application level styles.

Of course this could also be done with another `<Global>` element. Minimizing the API surface could reduce the risk of misuse.